README.md

```diff
@@ -87,18 +86,21 @@ And now it will hold the value `4`.
 
 Variables can be of many types: strings, arrays, numbers, dictionaries, and many more. The types of variables which refer only to a value (such as `NSInteger`) are referred to as "**primitives**". This is because, unlike the other variable types called "**objects**", primitives are unable to perform behaviors on the values that they reference. Subsequently, an object is a variable which can refer to one or many values that it can itself perform actions upon (these actions are called "**methods**"). 
 
-A string is a type of object which is used to store and manipulate text. In Objective-C, the base string class is `NSString`. We can create a string variable in much the same way as creating an integer variable.
+A string is a type of object which is used to store and manipulate text. While the string literal can be used in conjunction with `NSLog()` to print directly to the console:
```

Comment:
Hmm.. not a complete sentence. Maybe: "We've already seen string literals used in conjunction with `NSLog()` to print directly to the console:..."